### ISSUE TITLE
error  Parsing error: Unexpected token import

### ISSUE THREAD

**Issue by ilyaztsv** [vuejs/eslint-plugin-vue#204]
**Tell us about your environment**

* **ESLint Version: 4.8.0**
* **eslint-plugin-vue Version: 3.13.0**
* **Node Version: 7.5.0**

**Please show your full configuration:**

```
module.exports = {
  root: true,
  parserOptions: {
    sourceType: 'module',
    ecmaVersion: 8,
  },
  extends: [
    'standard',
    'plugin:vue/recommended' // or 'plugin:vue/base'
  ],
  "env": {
    "browser": true,
  },
  // add your custom rules here
  'rules': {
    "indent": ["error", 2],
    "semi": ["error", "always"],
    "space-before-function-paren": ["error", "never"],
    // allow paren-less arrow functions
    'arrow-parens': 0,
    // allow async-await
    'generator-star-spacing': 0,
    // allow debugger during development
    'no-debugger': process.env.NODE_ENV === 'production' ? 2 : 0,
    // trailing comma
    'comma-dangle': ['error', 'always-multiline'],
  }
}
```

**What did you do? Please include the actual source code causing the issue.**

```vue
<template>
  <div id="app">
  </div>
</template>

<script>
export default {
  name: 'app2',
  components: {
    'component1': () => import('./components/Component1.vue'),
  },
  data() {
    return {
      isShowComponent: false,
    };
  },
  methods: {
    showComponent: function(event) {
      this.isShowComponent = true;
    },
  },
};
</script>

<style scoped lang="scss">
</style>
```

**What did you expect to happen?**

**What actually happened? Please include the actual, raw output from ESLint.**
`
src/App.vue
  16:25  error  Parsing error: Unexpected token import
`

**Comment by ilyaztsv** [comment]
sorry for the issue. I missed `"parser": "babel-eslint"`
please, close the issue

**Comment by michalsnik** [comment]
No problem, glad you figured it out :)

**Comment by shmarts** [comment]
I seem to be having this issue when I use the updated `parserOptions: { parser: 'babel-eslint' } ` instead of the old `'parser': 'babel-eslint'` format, but appending the old option right above parserOptions fixes it. Is this meant to be happening?

**Comment by ilyaztsv** [comment]
@shmarts `parser` is not a property of `parserOptions`. It's separate property for config:

```
{
  "parser": "babel-eslint",
  "parserOptions": {
    "sourceType": "module",
    "allowImportExportEverywhere": false,
    "codeFrame": false
  }
}
```

**Comment by shmarts** [comment]
Hmm, a week ago I was getting an error when parser wasn't within parserOptions, but I've removed a couple of eslint related packages recently and it's gone now.

**Comment by bjornbos** [comment]
Actually, you do need to set it as a property of parserOptions, have a look at https://github.com/vuejs/eslint-plugin-vue#what-is-the-use-the-latest-vue-eslint-parser-error

**Comment by pxwee5** [comment]
Adding `"parser": "babel-eslint"` will cause babel to highlight the `template` tag in Single File Components.

**Comment by armano2** [comment]
https://vuejs.github.io/eslint-plugin-vue/user-guide/#what-is-the-use-the-latest-vue-eslint-parser-error

```diff
- "parser": "babel-eslint",
  "parserOptions": {
+     "parser": "babel-eslint",
      "ecmaVersion": 2017,
      "sourceType": "module"
  }
```

**Comment by hzsrc** [comment]
I don't use vue in a project, but I get the same issue: `Parsing error: Unexpected token import `
Then I add `eslint-plugin-vue` to my `.eslintrc` file, and the error got disappeared!
What a supprise!

**Comment by neorganic** [comment]
@hzsrc  Can you please share your `.eslintrc.json` content?

**Comment by conorbros** [comment]
@neorganic adding this fixed my issue:    ` parserOptions: { parser: 'babel-eslint' },`

**Comment by jerrychong25** [comment]
> https://vuejs.github.io/eslint-plugin-vue/user-guide/#what-is-the-use-the-latest-vue-eslint-parser-error
> 
> ```diff
> - "parser": "babel-eslint",
>   "parserOptions": {
> +     "parser": "babel-eslint",
>       "ecmaVersion": 2017,
>       "sourceType": "module"
>   }
> ```

This is working, thanks!
